src/ts/cosmosJourneyer.ts

```diff
@@ -314,6 +314,14 @@ export class CosmosJourneyer {
         await starSystemView.initAssets();
         starSystemView.resetPlayer();
 
+        if (!navigator.keyboard) {
+            await alertModal(
+                "Your keyboard layout could not be detected. The QWERTY layout will be assumed by default."
+            );
+        }
+
+        await alertModal("Loading Complete! Welcome to Cosmos Journeyer."); 
```

Comment:
You forgot to remove this loading complete modal. The loading complete message is already communicated to the player by the progress bar.